sklearn/linear_model/tests/test_bayes.py

```diff
@@ -200,6 +201,11 @@ def test_toy_ard_object():
     assert_array_almost_equal(clf.predict(test), [1, 3, 4], 2)
 
 
+# This test fails on some platforms.
+# See https://github.com/scikit-learn/scikit-learn/issues/16097
+#     https://github.com/scikit-learn/scikit-learn/issues/15420
+#     https://github.com/scikit-learn/scikit-learn/issues/15186
+@pytest.mark.skipif(True, reason="skipping test_ard_accuracy_on_easy_problem.")
```

Comment:
Shouldn't this go in a separate PR in master first? (just to keep things cleaner, I promis to review quickly :D)

Reply:
@ogrisel suggested to skip on the 0.22.X branch if I'm not mistaken.

We should probably not skip it on master but find out what's going wrong instead, no ?

Reply:
It seems to only fails on specific platform of BLAS or scipy version, not sure exactly, so it's more for safety that we want to skip it in this branch

Reply:
yeah, it just would mean that we probably will end up having to do the same thing when we release 0.23 (if it's not fixed by then)

Reply:
ok, I'll add the issues to the milestone, and we can keep it in this branch